### ISSUE TITLE
Failure adding using yarn

### ISSUE THREAD

**Issue by wweaver** [sjinks/node-inet-xtoy#56]
Is this still in early development? Should I be trying to use this package?

This is what I get when I try to add

```bash
 % yarn add inet_xtoy
yarn add v1.22.17
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
...
[4/4] 🔨  Building fresh packages...
[-/18] ⠂ waiting...
[16/18] ⠂ inet_xtoy
[17/18] ⠂ nodemon
[-/18] ⠂ waiting...
error .../node_modules/inet_xtoy: Command failed.
Exit code: 1
Command: prebuild-install --runtime napi --tag-prefix '' || node-gyp rebuild
Arguments:
Directory: .../node_modules/inet_xtoy
Output:
prebuild-install WARN install No prebuilt binaries found (target=1 runtime=napi arch=x64 libc= platform=darwin)
gyp info it worked if it ends with ok
gyp info using node-gyp@5.1.0
gyp info using node@14.18.1 | darwin | x64
gyp info find Python using Python version 3.9.8 found at "/usr/local/opt/python@3.9/bin/python3.9"
gyp info spawn /usr/local/opt/python@3.9/bin/python3.9
gyp info spawn args [
gyp info spawn args   '.../.nvm/versions/node/v14.18.1/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '.../node_modules/inet_xtoy/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '.../.nvm/versions/node/v14.18.1/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '.../Library/Caches/node-gyp/14.18.1/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=.../Library/Caches/node-gyp/14.18.1',
gyp info spawn args   '-Dnode_gyp_dir=.../.nvm/versions/node/v14.18.1/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=.../Library/Caches/node-gyp/14.18.1/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=.../node_modules/inet_xtoy',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CXX(target) Release/obj.target/inet_xtoy/main.o
In file included from ../main.cpp:1:
../../node-addon-api/napi.h:32:6: error: Exception support not detected.       Define either NAPI_CPP_EXCEPTIONS or NAPI_DISABLE_CPP_EXCEPTIONS.
    #error Exception support not detected. \
     ^
../main.cpp:14:2: error: cannot use 'throw' with exceptions disabled
        throw err;
        ^
../main.cpp:22:3: error: cannot use 'throw' with exceptions disabled
                throw Napi::TypeError::New(env, "Wrong number of arguments");
                ^
../main.cpp:26:3: error: cannot use 'throw' with exceptions disabled
                throw Napi::TypeError::New(env, "The argument is expected to be a string");
                ^
../main.cpp:50:3: error: cannot use 'throw' with exceptions disabled
                throw Napi::TypeError::New(env, "Wrong number of arguments");
                ^
../main.cpp:67:3: error: cannot use 'throw' with exceptions disabled
                throw Napi::TypeError::New(env, "The argument is expected to be a string or a Buffer");
                ^
../main.cpp:79:3: error: cannot use 'throw' with exceptions disabled
                throw Napi::TypeError::New(env, "The argument is of the wrong size");
                ^
7 errors generated.
make: *** [Release/obj.target/inet_xtoy/main.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (.../.nvm/versions/node/v14.18.1/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:400:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:282:12)
gyp ERR! System Darwin 20.6.0
gyp ERR! command ".../.nvm/versions/node/v14.18.1/bin/node" ".../.nvm/versions/node/v14.18.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd .../node_modules/inet_xtoy
```

yarn version: 1.22.17
node version: 14.18.1
python version: 3.9.1

I was trying to use this package because I can't find a nice tool that mimics the inet6_aton mysql function in straight javascript. https://www.geeksforgeeks.org/inet6_aton-function-in-mysql/

**Comment by sjinks** [comment]
I don't have a Mac to test on, but I will take a look

**Comment by sjinks** [comment]
Just published 1.2.0, could you please try it?

**Comment by wweaver** [comment]
It installed! I'll play around with it. Thanks!

```bash
% yarn add inet_xtoy
yarn add v1.22.17
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
...
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 7 new dependencies.
info Direct dependencies
└─ inet_xtoy@1.2.1
info All dependencies
├─ inet_xtoy@1.2.1
├─ mkdirp-classic@0.5.3
├─ node-abi@3.5.0
├─ node-addon-api@4.2.0
├─ prebuild-install@7.0.0
├─ simple-get@4.0.0
└─ tar-fs@2.1.1
✨  Done in 27.41s.
```
